src/MacroTools/BookSystem/IBook.cs

```diff
@@ -15,15 +16,54 @@ public interface IBook
     /// Determines whether or not the book is visible.
     /// </summary>
     public bool Visible { get; set; }
-    
+      
     /// <summary>
-    /// Fired when the exit button is clicked.
+    /// The name of the Book's launcher Button.
     /// </summary>
-    public OnClickAction OnClickExitButton { set; }
-    
+    public string Title { get; init; }
+
     /// <summary>
-    ///    The name of the Book's launcher Button.
+    /// A button for exiting of the book
     /// </summary>
-    public string Title { get; init; }
+    public Button ExitButton { get; set; }
+
+    /// <summary>
+    /// A button for launching of the book
+    /// </summary>
+    public Button LauncherButton { get; set; }
+
+    /// <summary>
+    /// A button for moving to the next page of the book
+    /// </summary>
+    public Button MoveNextButton { get; set; }
```

Comment:
I don't think we want to expose the setter on these; I can't imagine a scenario where it's valid for a consumer to arbitrarily modify the Next Button, even if it might be valid to modify what it does. Maybe we should just expose the Get?

LauncherButton seems valid to expose the Set.